File: e1_TwoSum/typescript/Main.ts
import { Solution } from './Solution';

function main() {
    const solution = new Solution();
    const nums = [2, 7, 11, 15];
    const target = 9;
    const result = solution.twoSum(nums, target);

    console.log("Input nums: [" + nums.join(", ") + "]");
    console.log("Input target: " + target);
    console.log("Output: [" + result.join(", ") + "]");
}

main();

File: e1_TwoSum/java/Main.java
package e1_TwoSum.java;

import java.util.Arrays;

public class Main {
    public static void main(String[] args) {
        Solution solution = new Solution();
        int[] nums = {2, 7, 11, 15};
        int target = 9;
        int[] result = solution.twoSum(nums, target);
        System.out.println("Input nums: " + Arrays.toString(nums));
        System.out.println("Input target: " + target);
        System.out.println("Output: " + Arrays.toString(result));
    }
}
